test/e2e/simple_test.go

```diff
@@ -27,8 +27,11 @@ func TestE2ESimple(t *testing.T) {
 	}
 
 	if os.Getenv("E2E_VERSION") != "" {
-		latestVersion = os.Getenv("E2E_VERSION")
+		versionsStr := os.Getenv("E2E_VERSION")
```

Comment:
Answering my own question from above, I think this contains all possible versions